.github/workflows/linux.yml

```diff
@@ -0,0 +1,45 @@
+name: linux
+
+on: push
```

Comment:
I've seen this on a lot of different PRs/repos. Why not `pull_request` as well??

Reply:
Do you really need pull_request if you have push since a push will happen for all pull_request at least once?

Reply:
Oh I believe that push runs CI on branches that aren't necessarily PRs. I've just noticed that my PRs to liveview-native-core didn't have CI because they're PRs across forks. I think that's due to the lack of pull_request in the "on" trigger.

Reply:
Add `pull_request` to `on` for all workflows.